tools/rdock/rdock.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>
+    <requirements>
+        <requirement type="package" version="2013.1">rDock</requirement>
+    </requirements>
+    <command>
+        <![CDATA[ 
+    	ln -s $inputmol2 inputmol2.mol2 && cat $inputprm | sed "s|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g" | sed "s|REF_MOL.*|REF_MOL ligand.sd|g" > new.prm; 

```

Comment:
All Galaxy variables that contain a dataset path (e.g. the ones associated to a `<param type="data">`) or a user-supplied string (`<param type="text">`) must be quoted using single quotes. So this line should be:
```
ln -s '$inputmol2' inputmol2.mol2 && cat '$inputprm' | sed 's|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g' | sed 's|REF_MOL.*|REF_MOL ligand.sd|g' > new.prm &&
```